### DISCUSSION TITLE
Requires WiFi or Ethernet / Cellular not supported?

### DISCUSSION THREAD

**Discussion by J-T-W** [tsightler/ring-mqtt#357] [Support Q&A] [answered]
Hey there - been using this HA add-on for a while and I had a question on availability of devices when the base station is on cellular instead of WiFi/Ethernet - is it expected that all devices/entities/sensors should be unavailable when the base is not connected to the local network?  I noticed this when the base dropped from WiFi for a while today and the devices all became unavailable.  Thought it might have been a connection thing, so had the add-on reauthenticate and the logs all show a successful connect.  Once the base was back on WiFi, all devices reappeared.

**Comment by sktaylortrash** [comment, marked as answer]
That is definitely the expected behaviour.  Even the notification you get from Ring when it drops to cellular tells you it will have limited features until the connection is re-established.

**Reply by J-T-W** [reply]
Thanks!

**Comment by tsightler** [comment]
Yep, the standard base stations basically just become dumb alarms when the data connection is down.  The cellular backup is used only as a way to provide emergency alarm notifications to the central monitoring station, all other features are effectively offline if the data connection is down.

On the other hand, the Ring Pro base station includes full Internet connection backup over cellular data, assuming you are willing to pay for a Ring Protect Pro subscription.  This even allows other devices in your home to stay connected if the primary Internet connection is down.
